packages/browser/test/integration/browsers.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   bs_android_6: {
     base: "BrowserStack",
     browser: "Android Browser",
-    device: "Samsung Galaxy Note 4",
+    device: "Google Nexus 6",
```

Comment:
Fixes this https://github.com/getsentry/sentry-javascript/runs/1594862346#step:6:57